docs/HowToFAQ/FAQModuleNames.md

```diff
@@ -0,0 +1,19 @@
+---
+title: How do I disambiguate module names?
+---
+
+## Question
+
+How do I disambiguate module names in an example like this:
+```dafny
+{% include_relative FAQModuleNames0.dfy %}
+```
+
+## Answer
+
+There is no direct syntax to do what is wanted here.
```

Comment:
Might be worth a link to the corresponding issues https://github.com/dafny-lang/dafny/issues/2493 and https://github.com/dafny-lang/dafny/issues/2493

Reply:
Added a link.